lib/mirage.ml

```diff
@@ -1912,6 +1912,8 @@ module Project = struct
     "open Lwt.Infix\n\
      let return = Lwt.return\n\
      let run = OS.Main.run"
+  let packages = [package "mirage"]
```

Comment:
Ha I think I understand the confusion now. This is just used for ocamlfind names, but I thought re-using the already existing `package` type would be useful (for instance to allow configuration plugins later on). Do you think I should just have a list of strings there?